Jenkinsfile

```diff
@@ -3,10 +3,10 @@ class Globals {
     static final String PROJECT = 'flexprep'
     static final String NEXUS_IMAGE_REPO_INTERN = 'docker-intern-nexus.meteoswiss.ch'
     static final String NEXUS_IMAGE_REPO_PUBLIC = 'docker-public-nexus.meteoswiss.ch'
-    static final String NEXUS_IMAGE_NAME_INTERN = 'docker-intern-nexus.meteoswiss.ch/numericalweatherpredictions/flexpart_ifs/flexprep'
-    static final String NEXUS_IMAGE_NAME_PUBLIC = 'docker-public-nexus.meteoswiss.ch/numericalweatherpredictions/flexpart_ifs/flexprep'
+    static final String NEXUS_IMAGE_NAME_INTERN = 'docker-intern-nexus.meteoswiss.ch/numericalweatherpredictions/dispersionmodelling/flexpart_ifs/flexprep'
+    static final String NEXUS_IMAGE_NAME_PUBLIC = 'docker-public-nexus.meteoswiss.ch/numericalweatherpredictions/dispersionmodelling/flexpart_ifs/flexprep'
```

Comment:
Any chance we could use flexpart-ifs and not flexpart_ifs as the image name? For conformity with flexpart-cosmo?